gradle/libs.versions.toml

```diff
@@ -104,43 +103,43 @@ hilt-compiler = { module = "com.google.dagger:hilt-android-compiler", version.re
 hilt-gradle-plugin = { module = "com.google.dagger:hilt-android-gradle-plugin", version.ref = "hilt" }
 
 # Google
-google-services-plugin = { module = "com.google.gms:google-services", version = "4.4.1" }
-google-playservices-auth = { module = "com.google.android.gms:play-services-auth", version = "21.1.0" }
+google-services-plugin = { module = "com.google.gms:google-services", version = "4.4.2" }
+google-playservices-auth = { module = "com.google.android.gms:play-services-auth", version = "21.2.0" }
 google-play-core = { module = "com.google.android.play:core", version = "1.10.3" }
 google-play-core-ktx = { module = "com.google.android.play:core-ktx", version = "1.8.1" }
-firebase-crashlytics = { module = "com.google.firebase:firebase-crashlytics", version = "18.6.4" }
-firebase-crashlytics-gradle-plugin = { module = "com.google.firebase:firebase-crashlytics-gradle", version = "2.9.9" }
+firebase-crashlytics = { module = "com.google.firebase:firebase-crashlytics", version = "19.0.1" }
+firebase-crashlytics-gradle-plugin = { module = "com.google.firebase:firebase-crashlytics-gradle", version = "3.0.1" }
 
 # AndroidX
 androidx-security = { module = "androidx.security:security-crypto", version = "1.0.0" }
-androidx-biometrics = { module = "androidx.biometric:biometric", version = "1.2.0-alpha05" }
+androidx-biometrics = { module = "androidx.biometric:biometric", version = "1.4.0-alpha01" }
```

Comment:
@ILIYANGERMANOV, I believe with the merge some of my changes were overridden. This needs to be reverted to `1.2.0-alpha05` unless we upgrade the compileSdk to 35.

Reply:
Updated ✅ Thanks!